run_tests.sh

```diff
@@ -5,7 +5,7 @@ script_location="$(readlink -f "$(dirname "$0")")"
 
 workspace_cargo_manifest="$script_location/Cargo.toml"
 
-cargo test --manifest-path "$workspace_cargo_manifest" --workspace --exclude e2e
+#cargo test --manifest-path "$workspace_cargo_manifest" --workspace --exclude e2e
```

Comment:
.

Reply:
was disabled for local testing. Will enable it